dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/OmegaRef/GeneratorSynchronousExtTfo_3E_INIT.mo

```diff
@@ -18,7 +18,7 @@ model GeneratorSynchronousExtTfo_3E_INIT "Synchronous machine with 3 windings -
   import Dynawo.Electrical.Machines;
 
   extends Machines.BaseClasses_INIT.BaseGeneratorVariables_INIT;
-  extends BaseClasses_INIT.BaseGeneratorSynchronousExt3E_INIT;
+  extends BaseClasses_INIT.BaseGeneratorSynchronousExt_3E_INIT;
```

Comment:
I have renamed BaseGeneratorSynchronousExt3E_INIT as BaseGeneratorSynchronousExt_3E_INIT, this change is unwarranted but inconsequential (even on dynawo-rte) : a future PR will rename it BaseGeneratorSynchronousExt3W_INIT.